frontend/src/main/java/org/abs_models/frontend/ast/CoreAbs.ast

```diff
@@ -50,12 +51,12 @@ ParametricPartialFunctionDecl : PartialFunctionDecl ::= Annotation* TypeParamete
 
 UnknownDecl: Decl; // Unknown declarations
 AmbiguousDecl: UnknownDecl ::= <Alternative:java.util.List<Decl>>;
-ClassDecl: BehaviorDecl ::= Annotation* Param:ParamDecl* ImplementedInterfaceUse:InterfaceTypeUse* TraitUse* [InitBlock] RecoverBranch:CaseBranchStmt* Field:FieldDecl* Method:MethodImpl*;
+ClassDecl: BehaviorDecl ::= Annotation* Param:ParamDecl* ImplementedInterfaceUse:InterfaceTypeUse* TraitUse* [InitBlock] RecoverBranch:CaseBranchStmt* Field:FieldDecl* Method:MethodImpl* <Base:boolean> <Relative:boolean> ;
```

Comment:
What is the meaning of the `Base` and `Relative` flags; could they get more descriptive names?  Ditto for `InterfaceDecl`

Reply:
The grammar in `ABS.xtext` permits a class / interface to be both `base` and `relative`; is this valid?

Reply:
A class is `base` if it is the base for delta operations (i.e., it is modified by a delta) and it is `relative` if it is not modified by a delta, but references a `base` class. `base` implies `relative´ right now, but I want to have the option to change it once we have more feedback.